src/styles/global.css

```
@@ -640,9 +642,11 @@
 
   /* --- Bubble Chart (BubbleChart) --- */
   --em-bubblechart-point-opacity: 0.7;
```

Stylelint (17.9.0)

[error] Duplicate custom property "--em-bubblechart-point-opacity" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)